data/io.elementary.wingpanel.bluetooth.appdata.xml.in

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 elementary, Inc. <contact@elementary.io> -->
+<component type="addon">
+  <id>io.elementary.wingpanel.bluetooth</id>
+  <extends>io.elementary.wingpanel</extends>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-2.1</project_license>
+  <name>Wingpanel Bluetooth Indicator</name>
+  <summary>Provides an indicator in Wingpanel for managing Bluetooth connections</summary>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/elementary/wingpanel-indicator-bluetooth/master/data/screenshot.png</image>
+    </screenshot>
+  </screenshots>
+  <provides>
+    <library>libbluetooth.so</library>
+  </provides>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
```

Comment:
Hmm, I don't think we need to add this content rating since this is an official component. For example, [the .appdata.xml file in Switchboard Mouse &amp; Touchpad](https://github.com/elementary/switchboard-plug-mouse-touchpad/blob/master/data/io.elementary.switchboard.mouse-touchpad.appdata.xml.in) or other .appdata files for Switchboard Plugs does not seem to include these elements.

Reply:
At least the new desktop applications, like side-load and tasks, have OARS data in their AppData file and I would consider that as official components of elementary :stuck_out_tongue: So I don't think that would necessary be a reason to leave it out of "addons".

But I'm happy to leave it out, it can be added later in a separate PR if it's wanted.

Reply:
I asked Cassidy and according to him, we may want to add OARS metadata to all components.